lib/connect.js

```diff
@@ -690,6 +690,7 @@ function relayConnection (c, relayThrough, payload, hs) {
     publicKey = relayThrough
     token = c.relayToken
   }
+  c.encryptedSocket.emit('relaying', { relayThrough: publicKey })
```

Comment:
just emit(relaying, publicKey) is fine

Reply:
ah nvm, it pairs nicely with the other one, would add remotePublicKey: c.remotePublicKey also here so its the same object emitted always between the conn and the server